2009-03-01  Aurelien Jarno  <aurelien@aurel32.net>

	* nss/nsswitch.c (lock_nsswitch, unlock_nsswitch): use braces
	instead of brackets.

--- a/nss/nsswitch.c	(revision 8009)
+++ b/nss/nsswitch.c	(working copy)
@@ -84,8 +84,8 @@
 
 __libc_lock_define_initialized (static, lock)
 
-#define lock_nsswitch (__libc_lock_lock (lock))
-#define unlock_nsswitch (__libc_lock_unlock (lock))
+#define lock_nsswitch {__libc_lock_lock (lock)}
+#define unlock_nsswitch {__libc_lock_unlock (lock)}
 
 #if !defined DO_STATIC_NSS || defined SHARED
 /* String with revision number of the shared object files.  */
